pkg/semodule/policycoreutils/policycoreutils.go

```diff
@@ -47,16 +47,23 @@ func (smt *SEModulePcuHandler) Install(modulePath string) error {
 }
 
 func (smt *SEModulePcuHandler) List() ([]string, error) {
-	out, err := runSemodule("-l")
+	out, err := runSemodule("-lfull")
 	if err != nil {
 		smt.logger.Error(err, "Listing policies")
 		return nil, seiface.ErrList
 	}
-	return strings.Split(string(out), "\n"), nil
+	modules := make([]string, 0)
+	for _, line := range strings.Split(string(out), "\n") {
+		module := strings.Split(line, " ")
+		if module[0] == "350" {
+			modules = append(modules, module[1])
+		}
+	}
+	return modules, nil
```

Comment:
I see, so we need to parse out the rest of the modules. Is it possible to target listing only modules of a certain priority?

Reply:
Currently it's not possible. But I'd like to have this or similar feature in the 3.5 selinux userspace release